http/src/main/resources/reference.conf

```diff
@@ -215,23 +215,23 @@ azure {
      # See https://github.com/broadinstitute/cromwhelm/blob/main/terra-batch-libchart/templates/_reverse-proxy.tpl#L81-L114
      # for list of services.
      services = [
```

Comment:
Figured I can use this array of services to control which are enabled/disabled in cromwhelm. This array also populates the `proxyUrls` in the Leo `getApp` JSON response.